### PR TITLE
[DOCS-6481] Consistency updates for ACS docs - Using Share

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: content-services/5.2/using/share.md [hunk 1]
```diff
@@ -1,297 +1,129 @@
...
--   **Site Search**
-
-    Lets you quickly search all sites you belong to
-
--   **Saved Search**
-
-    Displays the results of a specific search that you configure
-
--   **My Document Workspaces**
+> **Note:** If you've [set your home page]({% link content-services/5.2/using/dashboard.md %}#sethomepage) then when you click **Home** that page will open.
```
Comment (gravitonian) [inline content-services/5.2/using/share.md:60]: Incorrect in-page link

**Review by gravitonian — CHANGES_REQUESTED** [review]
All pages looks alright, just one in-page for 5.2 needs fixing.

File: content-services/5.2/using/share.md (hunk 1 above)
Comment (anxumalo) [reply content-services/5.2/using/share.md:60]: Fixed the related link in `content-services/5.2/using/dashboard.md`

**Review by gravitonian — APPROVED** [review]
Looks good!

**Pull request merged**